### PR TITLE
slow release of batched restore

### PR DESCRIPTION
**Pull request by snopoke**
@czue

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by czue** [comment]
real build fail?

```
======================================================================
FAIL: test_multiple_batches_restore (casexml.apps.phone.tests.test_batched_mode.BatchRestoreTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/local/lib/python2.7/site-packages/mock.py", line 1224, in patched
    return func(*args, **keywargs)
  File "corehq/ex-submodules/casexml/apps/phone/tests/test_batched_mode.py", line 48, in test_multiple_batches_restore
    self.assertEqual(restore_config.num_batches, 4)
AssertionError: None != 4
```

**Comment by dimagimon** [comment]
:+1: No lint errors found.

File: corehq/toggles.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ class PredicatablyRandomToggle(StaticToggle):
     """
 
     def __init__(self, slug, label, namespace, randomness):
-        super(PredicatablyRandomToggle, self).__init__(slug, label, [namespace])
+        super(PredicatablyRandomToggle, self).__init__(slug, label, list(namespace))
```
Comment (snopoke) [inline corehq/toggles.py]: sneaky bugger

**Comment by snopoke** [comment]
@czue :white_check_mark:

File: corehq/toggles.py (hunk 1 above)
Comment (czue) [inline corehq/toggles.py]: oops. sorry about that.

**Pull request merged**